### PR TITLE
Actually updating the tips file

### PR DESCRIPTION
**Pull request by jackowski626**
This file actually doesn't regenerate as long as the old one stays in config. Might as well update this one. Ran this file through Grammarly too, in order to reduce grammar and spelling errors

### CODE REVIEW

File: config/Betterloadingscreen/tips/en_US.txt [hunk 1]
```diff
@@ -1,66 +1,150 @@
...
+Got a question? Join our Discord server: discord.com/invite/EXshrPV
 Don't give ideas to 0lafe
-Don't feed your machines after midnight
+Don't feed your machines after midnight!
 Make sure you have installed a backup mod
 Material tiers play a role when breaking pipes
 If a machine catches fire, it can explode
-Adding water to an empty but hot Boiler will cause an explosion
-Avoid eldritch obelisks
+Adding water to an empty but hot Boiler will make a loud bang!
```
Comment (KiloJoel) [inline config/Betterloadingscreen/tips/en_US.txt:14]: I would prefer the original here, less ambiguous.

File: config/Betterloadingscreen/tips/en_US.txt [hunk 2]
```diff
@@ -1,66 +1,150 @@
...
-Farm Glowflowers for glowstone
-Making steel armour? Check out the composite chestplate
-Adventurer's backpack: integrated crafting grid, bed and fluid storage
-Beware of cable power loss
-Machines that get a higher voltage than they can handle explode
+Shift-right-clicking with a wrench on a fluid pipe makes it input-only
+The Bending Machine makes plates more efficiently
+Multiblocks can share walls
+You can't use the front side of machines
+Disable a machine with a Soft Mallet if it can not finish a recipe
```
Comment (KiloJoel) [inline config/Betterloadingscreen/tips/en_US.txt:33]: change "can not" to "can't"

File: config/Betterloadingscreen/tips/en_US.txt [hunk 3]
```diff
@@ -1,66 +1,150 @@
...
+Don't go out at night! Mobs can have up to 300 HP
+Steam is hot! It will burn Wooden Fluid Pipes
+Don't break rubber trees without a Treetap
+Make Muffler Moint Mup (Point up)
+Your EBF won't work if the muffler doesn't point to the sky!
+Eating too many of the same food will make you hate it!
+Abandon all vanilla knowledge all ye who enter here
+With the power of Duct Tape, I solved all of your maintenance problems!
+When you open a Twilight Portal, you'll get hit by lightning
+#Sometimes it's better to flee than fight some mobs
```
Comment (KiloJoel) [inline config/Betterloadingscreen/tips/en_US.txt:106]: remove #

File: config/Betterloadingscreen/tips/en_US.txt [hunk 4]
```diff
@@ -1,66 +1,150 @@
...
+Sometimes the best combat strategy is "Nigerundayooo!"
+You can power machines by putting a battery in their energy slots
+You can charge your jetpack by putting it in the Energy slot of LV machines
+Be prepared to take hours to craft a single machine!
+Oil + Thunder = Fire. Remember to put cobblestone over your oil wells!
+Nuclear Reactor configs give 10x the energy they give in IC2!
+Remember to plant cotton - you'll need a ton of it
+It's aluminIum, not aluminum
+#Sometimes it's better to buy than to craft
```
Comment (KiloJoel) [inline config/Betterloadingscreen/tips/en_US.txt:115]: is # meant to be there? idk

File: config/Betterloadingscreen/tips/en_US.txt [hunk 5]
```diff
@@ -1,66 +1,150 @@
...
+Life's all about opening issues, not closing them - Rongmario
+Batteries can be charged and discharged in the same slot
+3D-printed Lave 12 is the future - Sampsa
+Don't hold machines in your hotbar when playing with cables
+Was making HV connection. Missclicked and placed MV engraver - Lewissaber
+Destroyed once a Tier 1 rocket by accidentally pressing shift - Regian
+Visit r/feedthemems!
+I WANT ROCKET I WANT MOON - St4o
+Shift + Right Click on a GregTech Cover to open the menu
+Looking for Infused Stones? Try putting Amber in an Ore Finder Wand
```
Comment (KiloJoel) [inline config/Betterloadingscreen/tips/en_US.txt:150]: I suggest adding "Remove plates from machines using a crowbar"

File: config/Betterloadingscreen/tips/en_US.txt [hunk 6]
```diff
@@ -1,66 +1,150 @@
...
+How many circuits have you crafted in your life?
+Journeymap: press [J]
+Not by Vazkii
+sqrt(-1) loves you!
+(MultiMC > Curse) && (Technic > Curse): true
+All you had to do was tho follow the damn questline, CJ!
+Click on an item in the quest book to show its recipe in NEI
+Why not visit GregTech? We have... microcrafting
+Cables are | || || |_ y
+Varying foods gives hearts!
```
Comment (KiloJoel) [inline config/Betterloadingscreen/tips/en_US.txt:133]: maybe "Eating new foods gives hearts!"

File: config/Betterloadingscreen/tips/en_US.txt (hunk 2 above)
Comment (bartimaeusnek) [reply config/Betterloadingscreen/tips/en_US.txt:33]: I would prefer "can not" since "can't" is colloquial.

Comment (KiloJoel) [reply config/Betterloadingscreen/tips/en_US.txt:33]: "cannot" would be better if staying with less colloquial

Comment (jackowski626) [reply config/Betterloadingscreen/tips/en_US.txt:33]: Tips of this kind tend to be colloquial. Think of the splash messages in minecraft

**Comment by jackowski626** [comment]
ok, gonna change some stuff

**Review by Dream-Master — APPROVED** [review]

File: config/Betterloadingscreen/tips/en_US.txt (hunk 2 above)
Comment (basdxz) [reply config/Betterloadingscreen/tips/en_US.txt]: Can't makes most sense in this context.

File: config/Betterloadingscreen/tips/en_US.txt (hunk 5 above)
Comment (basdxz) [reply config/Betterloadingscreen/tips/en_US.txt]: r/feedthemems -> r/feedthememes
Dropped the e

**Comment by Dream-Master** [comment]
@basdxz feel free to edit the file and push it. Thanks

**Pull request merged**